server/parsing_lib/log_parser.py

```diff
@@ -8,61 +8,128 @@
 
 
 class LogParser(object):
-    """ Handles parsing of all logs from the Mobile API. """
+    """ Handles parsing of all logs from the Mobile API.
+
+    Attributes:
+        parser_info: info required by the parser that is specific to each OS
+    """
+    parser_info = {
```

Comment:
I actually really like this. I could even hide that behind a set of static methods which figure out which class to call. I've added an issue for it: #66.